README.md

```diff
@@ -426,6 +427,84 @@ let data = SavedData.read('user-value-key'); //data will be either the value in
 SavedData.remove('user-value-key'); //removes the value from both local and session storage.
 ```
 
+Following is an example of interacting with [IndexedDB](https://developer.mozilla.org/en-US/docs/Web/API/IndexedDB_API) through the UserData class
```

Comment:
Is this the same documentation that is being added [here](https://github.com/SpringRoll/SpringRoll/pull/294)? We should pick one spot that makes the most sense so we don't need to keep things in sync in lots of places.